src/app/booking/[id]/page.tsx

```diff
@@ -0,0 +1,44 @@
+"use client";
+
+import Sidebar from "@/app/components/sidebar/sidebar";
+import Booking from "@/app/components/booking";
+import { useEffect, useState } from "react";
+
+const fetchBookingDetails = async (id: string) => {
+  const res = await fetch(`/api/bookingDetails/${id}`)
+  const data = await res.json()
+
+  return data
+}
```

Comment:
I understand what you are doing, you are calling Next as a backend so it calls notion

No need,  frontend can call Notion directly (we will avoid Notion as a backend as long Stripe doesn't force us)